packages/react/src/components/FileUploader/index.stories.js

```diff
@@ -19,13 +18,18 @@ stories.add("Playground", () => (
     onChange={action("onChange")}
     isLoading={boolean("isLoading", false)}
     hasError={boolean("hasError", false)}
+    multiple={boolean("has multiple files", true)}
```

Comment:
🌷`Has multiple files?`, `Has error`, `Is loading?`, etc.. At least I've been doing it this way in other components :)

Reply:
Yep, good point.
Maybe we should have guidelines for that too 🤔